packages/salesforcedx-utils-vscode/src/cli/commandBuilder.ts

```diff
@@ -42,7 +42,11 @@ export class CommandBuilder {
   }
 
   public withArg(arg: string): CommandBuilder {
-    this.args.push(arg);
+    if (arg === '--json') {
+      this.withJson();
```

Comment:
I noticed a bunch of calls like `withArg('--json')`, so I figured I'd add this here in case someone accidentally uses withArg in the future, instead of withJson.